cytotable/convert.py

```diff
@@ -1054,7 +1034,6 @@ def _to_parquet(  # pylint: disable=too-many-arguments, too-many-locals
         _infer_source_group_common_schema,
         _join_source_chunk,
         _prepend_column_name,
-        _return_future,
         _source_chunk_to_parquet,
     )
```

Comment:
Are these import statements still required to be within the function if `to_parquet` is no longer a parsl app?

Reply:
Thanks this is a great point! I believe you're correct that these imports may now be removed.